include/fmt/format-inl.h

```diff
@@ -8,13 +8,15 @@
 #ifndef FMT_FORMAT_INL_H_
 #define FMT_FORMAT_INL_H_
 
-#include <algorithm>
-#include <cerrno>  // errno
-#include <climits>
-#include <cmath>
-#include <exception>
+#ifndef FMT_IMPORT_STD
+#  include <algorithm>
+#  include <cerrno>  // errno
```

Comment:
`errno` is often a macro so this include should be moved outside of the ifndef block.